.github/workflows/cpp_cron.yml

```diff
@@ -144,7 +147,7 @@ jobs:
   ubuntu-cmake32:
     name: AMD64 Ubuntu 18.04 C++ CMake 3.2
     runs-on: ubuntu-latest
-    if: github.event_name == 'push'
+    if: ${{ !contains(github.event.pull_request.title, '[skip ci]') }}
```

Comment:
Do we need this change?

Reply:
It is already in cron, so we don't need to filter on the event.